app/Repositories/ProgramRepository.php

```diff
@@ -1,6 +1,7 @@
 <?php 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/ProgramRepository.php</br>Whitespace found at end of line